arena.js

```diff
@@ -0,0 +1,54 @@
+class Arena {
+  constructor(name){
+    this.name = capitalizeFirstLetter(name);
+    this.gladiators = []
+  }
+    addGladiator(x){
+      if (this.gladiators.length < 2){
+      this.gladiators.push(x)
+    }else{
+      alert("You can't have more than two gladiators in the arena")
+    }
+  }
+  fight(){
+    var Glad1 = this.gladiators[0].weapon;
+    var Glad2 = this.gladiators[1].weapon;
+    if ((Glad1 == "Trident" && Glad2 == "Spear") ||
+       (Glad1 == "Spear" && Glad2 == "Club") ||
+       (Glad1 == "Club" && Glad2 == "Trident")){
```

Comment:
this is great, but can you think of any way to abstract these winning conditions so as to avoid these many conditionals?